scripts/k8s/deployer.sh

```diff
@@ -87,10 +89,11 @@ while [ "$#" -gt 0 ]; do
     --delay)
       DELAY="true"
       shift;;
-    --label)
-      LABEL="true"
+    --label-io-engine)
```

Comment:
you've missed the "--mayastor" arg parsing here?

Reply:
added